card/lib/card/migration.rb

```diff
@@ -102,6 +102,13 @@ def merge_cards names_or_keys
       Card::Migration::Import.new(data_path).merge only: names_or_keys
     end
 
+    def merge_pristine_cards names_or_keys
+      names_or_keys = Array(names_or_keys)
+
+      pristine = names_or_keys.select { |n| !Card.exists?(n) || Card.fetch(n)&.pristine? }
```

Comment:
isn't the `!Card.exists?` unnecessary  here?

Reply:
Oh, I guess you'd need to fetch with "new", eg:
`Card.fetch(n, new: {}).pristine?`

Reply:
Yes, I noticed that that would work while I was fixing this but it didn't feel like an improvement.
(the real issue was the missing ! on the select call)